src/controllers/__tests__/events.test.js

```diff
@@ -65,7 +65,7 @@ const ngos = [
 
 const volunteers = [
   {
-    _id: '1',
+    id: '1',
     firstName: 'Dilara',
     lastName: 'Fırtına',
```

Comment:
test passed successfully, the Last name includes  The character U+0131 "ı" could be confused with the character U+0069 "i", which is more common in source code